config/localTemplate.go

```diff
@@ -44,13 +44,13 @@ type Local struct {
 	// for an existing parameter. This field tag must be updated any time we add a new version.
 	Version uint32 `version[0]:"0" version[1]:"1" version[2]:"2" version[3]:"3" version[4]:"4" version[5]:"5" version[6]:"6" version[7]:"7" version[8]:"8" version[9]:"9" version[10]:"10" version[11]:"11" version[12]:"12" version[13]:"13" version[14]:"14" version[15]:"15" version[16]:"16" version[17]:"17" version[18]:"18" version[19]:"19" version[20]:"20" version[21]:"21" version[22]:"22" version[23]:"23" version[24]:"24" version[25]:"25" version[26]:"26" version[27]:"27" version[28]:"28" version[29]:"29" version[30]:"30" version[31]:"31" version[32]:"32" version[33]:"33"`
 
-	// Archival nodes retain a full copy of the block history. Non-Archival nodes will delete old blocks and only retain what's need to properly validate blockchain messages (the precise number of recent blocks depends on the consensus parameters. Currently the last 1321 blocks are required). This means that non-Archival nodes require significantly less storage than Archival nodes. Relays (nodes with a valid NetAddress) are always Archival, regardless of this setting. This may change in the future. If setting this to true for the first time, the existing ledger may need to be deleted to get the historical values stored as the setting only effects current blocks forward. To do this, shutdown the node and delete all .sqlite files within the data/testnet-version directory, except the crash.sqlite file. Restart the node and wait for the node to sync.
+	// Archival nodes retain a full copy of the block history. Non-Archival nodes will delete old blocks and only retain what's need to properly validate blockchain messages (the precise number of recent blocks depends on the consensus parameters. Currently the last 1321 blocks are required). This means that non-Archival nodes require significantly less storage than Archival nodes.  If setting this to true for the first time, the existing ledger may need to be deleted to get the historical values stored as the setting only effects current blocks forward. To do this, shutdown the node and delete all .sqlite files within the data/testnet-version directory, except the crash.sqlite file. Restart the node and wait for the node to sync.
```

Comment:
Rather than just removing the note about Relays, should it say something about the 22k retention policy?

Reply:
I leaned no, as that isn't in any code. The 22k is solely in an algocfg profile for relays.

Reply:
It seems weird to have a node generate and return catchpoints, but not retain enough blocks to allow a node to use that catchpoint to catchup.  I'd be in favor, though I don't feel strongly, of making the node retain `2*catchpointInterval+fudge` if it's a catchpoint node. (Invisibly - just as we retain at least 1321 blocks even if you set the retention policy to, say, 500 blocks.

Reply:
https://github.com/algorand/go-algorand/blob/master/cmd/algocfg/profileCommand.go#L69 specifically